src/commands/deployment/execute.rs

```diff
@@ -379,113 +385,74 @@ pub async fn handle_execute(
         let progress_bar = new_spinner_progress_bar();
         progress_bar.set_message("Fetching available build artifacts ...");
 
-        let cd_pipeline_data = client
-            .fetch_cd_pipeline(
+        let cd_pipeline_data: Option<CdPipelineWithBuilds>;
+        let mut pipeline_type = PipelineType::Github;
+
+        let github_cd_pipeline = get_github_cd_pipeline(
+            &mut client,
+            &current_application,
+            &selected_namespace.to_lowercase(),
+            &selected_version.to_lowercase(),
+        )
+        .await;
+
+        if github_cd_pipeline.is_ok() {
```

Comment:
The error from `get_github_cd_pipeline` is not handled.

Reply:
Its intentional, reason being is that when github endpoint fails to get the pipeline weather pipeline is not setup/not_found we fallback and fetch from jenkins.

Reply:
are we want to fallback to jenkins pipeline for any error message from github endpoint?